test/e2e/cypress/support/commands.js

```diff
@@ -85,6 +85,25 @@ Cypress.Commands.add('updateApiKeyExpiration', (apiKeyExpiration) => {
   });
 });
 
+Cypress.Commands.add('preloadTestData', () => {
```

Comment:
nitpick: can we put a docstring comment to this?
Something like:
```
Preload required test data. It must run photofinish scenario twice as the order of sent payloads is relevant and the tests require a fully loaded scenario which only happens when the scenario is sent in the second time
```
At the end, this is a handicap we have and maybe a temporary solution.
```